src/main/js/portal/main/components/preview/PreviewTimeSerie.tsx

```diff
@@ -158,15 +158,17 @@ export default class PreviewTimeSerie extends Component<OurProps, OurState> {
 
 		const showChartTypeControl = !previewExcludeChartType.datasets.includes(preview.item.dataset!);
 
-		const yParam = yAxis ? `&y=${yAxis}` : '';
-		const y2Param = y2Axis ? `&y2=${y2Axis}` : '';
-		const legendLabelsParams = legendLabels ? `&legendLabels=${legendLabels}` : '';
-		const legendLabelsY2Params = y2Axis && legendLabels ? `&legendLabelsY2=${legendLabels}` : '';
-		const typeParam = type ? `&type=${type}` : '';
-		const iframeUrl = `${window.document.location.protocol}//${window.document.location.host}${iFrameBaseUrl}?objId=${objIds}&x=${xAxis}${yParam}${y2Param}${typeParam}&linking=${linking}${legendLabelsParams}${legendLabelsY2Params}`;
-
-		if (!preview)
-			return null;
+		const iframeParams = [`?objId=${objIds}`];
+		iframeParams.push(`&x=${xAxis}`);
+		iframeParams.push(yAxis ? `&y=${yAxis}` : '');
+		iframeParams.push(y2Axis ? `&y2=${y2Axis}` : '');
+		iframeParams.push(`&linking=${linking}`);
+		iframeParams.push(legendLabels ? `&legendLabels=${legendLabels}` : '');
+		iframeParams.push(y2Axis && legendLabels ? `&legendLabelsY2=${legendLabels}` : '');
+		iframeParams.push(type ? `&type=${type}` : '');
+
+		const iframeUrl = `${window.document.location.protocol}//${window.document.location.host}`
+							+ `${iFrameBaseUrl}${iframeParams.join('')}`;
```

Comment:
Would it be worth splitting it in 3-4 lines?
We usually use spaces for alignment.